components/speedreader/BUILD.gn

```diff
@@ -51,3 +46,25 @@ source_set("speedreader") {
     "//url",
   ]
 }
+
+source_set("unit_tests") {
+  testonly = true
+  if (enable_speedreader) {
+    sources = [
+      "//brave/components/speedreader/rust/ffi/speedreader_unittest.cc",
+      "//brave/components/speedreader/speedreader_throttle_unittest.cc",
+      "//brave/components/speedreader/speedreader_util_unittest.cc",
+    ]
+
+    deps = [
+      "//base",
+      "//brave/components/speedreader",
+      "//brave/components/speedreader/common",
+      "//chrome/browser",
```

Comment:
it is sort of illegal to have a dep on chrome here, that's why unittests are listed in the root BUILD.gn
at some point we should add our own `components_unittests` binary to properly handle this, wdyt @bridiver ?

Reply:
Right, I think tihs is layer violation.
If this test needs chrome layer dependency, I think this test target could be moved to /brave/browser/speedreader:unit_test target?

Reply:
we need to fix this @keur